zeebe/exporters/elasticsearch-exporter/src/main/java/io/camunda/zeebe/exporter/ElasticsearchClient.java

```diff
@@ -163,7 +164,8 @@ public boolean shouldFlush() {
    * @return true if request was acknowledged
    */
   public boolean putIndexTemplate(final ValueType valueType) {
-    final String templateName = indexRouter.indexPrefixForValueType(valueType);
+    final String templateName =
+        indexRouter.indexPrefixForValueType(valueType, VersionUtil.getVersionLowerCase());
```

Comment:
❓ Here we can use the VersionUtil, because we create the new template right? For the new version of the Exporter?

Reply:
Yes, this should be fine.